studio/components/LanguageSelector.tsx

```diff
@@ -0,0 +1,151 @@
+import { Box, Button, Card, Checkbox, Flex, Text, useTheme } from "@sanity/ui";
+import React from "react";
+import { PatchEvent, set } from "sanity";
+
+import {
+  Language,
+  supportedLanguages,
+} from "internationalization/supportedLanguages";
+
+interface LanguageSelectorProps {
+  value?: Language[];
+  onChange: (event: PatchEvent) => void;
+}
+
+const colorMap = {
```

Comment:
Should this be moved to a shared utils/styling file?

Reply:
Hm good question. This is more specific to this selector, but it's definitely worth looking into in the issue created for shared color values.